pkgs/by-name/fo/foks/package.nix

```
@@ -0,0 +1,76 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  pcsclite,
+  pkg-config,
+  stdenv,
+}:
+let
+  templFoks = buildGoModule rec {
+    pname = "templFoks";
+    version = "0.3.833";
+
+    src = fetchFromGitHub {
+      owner = "a-h";
+      repo = "templ";
+      rev = "v${version}";
+      hash = "sha256-4K1MpsM3OuamXRYOllDsxxgpMRseFGviC4RJzNA7Cu8=";
+    };
+
+    vendorHash = "sha256-OPADot7Lkn9IBjFCfbrqs3es3F6QnWNjSOHxONjG4MM=";
+
+    subPackages = [ "cmd/templ" ];
+
+    env.CGO_ENABLED = 0;
+
+    ldflags = [
+      "-s"
+      "-w"
+      "-extldflags -static"
+    ];
+    meta = {
+      description = "Language for writing HTML user interfaces in Go";
+      homepage = "https://github.com/a-h/templ";
+      license = lib.licenses.mit;
+      mainProgram = "templ";
+      maintainers = with lib.maintainers; [
+        luleyleo
+      ];
+    };
+  };
+in
```

You don't need to re-declare the entire package. Use overrides

```suggestion
{
  lib,
  buildGoModule,
  fetchFromGitHub,
  pcsclite,
  pkg-config,
  stdenv,
  templ,
}:
let
  templFoks = templ.overrideAttrs (old: {
    pname = "templ-foks";
    version = "0.3.833";

    src = old.src.override {
      hash = "sha256-4K1MpsM3OuamXRYOllDsxxgpMRseFGviC4RJzNA7Cu8=";
    };

    vendorHash = "sha256-OPADot7Lkn9IBjFCfbrqs3es3F6QnWNjSOHxONjG4MM=";
  });
in
```